CHANGELOG-5.0.md

```diff
@@ -266,7 +266,7 @@ To get the diff between two versions, go to https://github.com/symfony/symfony/c
  * bug #34802 [Security] Check UserInterface::getPassword is not null before calling needsRehash (dbrekelmans)
  * bug #34788 [SecurityBundle] Properly escape regex in AddSessionDomainConstraintPass (fancyweb)
  * bug #34859 [SecurityBundle] Fix TokenStorage::reset not called in stateless firewall (jderusse)
- * bug #34827 [HttpFoundation] get currently session.gc_maxlifetime if ttl doesnt exists (rafaeltovar)
+ * bug #34827 [HttpFoundation] get currently session.gc_maxlifetime if ttl doesn't exists (rafaeltovar)
```

Comment:
Please revert this change. The file is generated automatically based on the used commit messages.

Reply:
If i revert this `fabbot.io` is on error

Reply:
It’s fine to revert, it’s a false positive 👍🏻

Reply:
@fabpot could you exclude these auto-generated files from the fabbot check ?

Reply:
ok i have reverted the file thanks for your feedbacks